src/main.rs

```diff
@@ -37,6 +37,8 @@ struct Args {
     /// Number of times to inline leaves
     #[clap(long, default_value = "0")]
     serialize_n_inline_leaves: usize,
+    #[clap(long)]
+    dont_print_messages: bool,
```

Comment:
Optional: should this be called `quiet` or `silent`?

Reply:
I renamed it to `--no-messages` to make it a bit shorter but also to keep it explicit about what part of the output it's disabling.

Reply:
Looks good, thanks!